src/features/posts/post.js

```diff
@@ -0,0 +1,176 @@
+import React from "react";
+import { Comments } from "../comments/comments";
+import { useNavigate } from "react-router-dom";
+import {
+  Card,
+  CardActions,
+  CardHeader,
+  CardContent,
+  IconButton,
+  Button,
+  ButtonGroup,
+  Box,
+  CardMedia,
+  Typography,
+  Collapse,
+} from "@mui/material";
+import ShareIcon from "@mui/icons-material/Share";
+import ChatBubbleIcon from "@mui/icons-material/ChatBubble";
+import BookmarkBorderIcon from "@mui/icons-material/BookmarkBorder";
+import MoreHorizIcon from "@mui/icons-material/MoreHoriz";
+import ThumbUpOffAltIcon from "@mui/icons-material/ThumbUpOffAlt";
+import ThumbDownOffAltIcon from "@mui/icons-material/ThumbDownOffAlt";
+import { relativeDays } from "../../components/helperFunctions";
+
+export const Post = ({ selectedSubreddit, post, fullVersion = false }) => {
```

Comment:
This component can probably be broken into smaller ones, like the whole part with thumb up/down or the bottom bar with comments.

This could allow for an even better architecture:
- PostItem (to be displayed in posts list on subreddits)
- FullPost (to be displayed on full post page)
This could make the code easier to maintain